src/lib/features/instance-stats/instance-stats-service.ts

```diff
@@ -165,14 +163,9 @@ export class InstanceStatsService {
         this.clientMetricsStore = clientMetricsStoreV2;
     }
 
-    async refreshStatsSnapshot(): Promise<void> {
+    async refreshAppCountSnapshot(): Promise<void> {
         try {
-            this.snapshot = await this.getStats();
-            const appCountReplacement = {};
-            this.snapshot.clientApps?.forEach((appCount) => {
-                appCountReplacement[appCount.range] = appCount.count;
-            });
```

Comment:
Avoid querying all statistics just to get the appCount. Note that the property `this.snapshot` was never used.